### DISCUSSION TITLE
The latest windows 64 bit download is getting blocked by windows defender

### DISCUSSION THREAD

**Discussion by mikeboysen** [lencx/ChatGPT#578] [Other]
It's telling me it detected Contebrew.A!ml and no matter what I do it will not download. v12 was updated on my computer yesterday and was running, then I found it uninstalled this morning.

**Comment by lencx** [comment]
ref: https://github.com/lencx/ChatGPT/issues/559#issuecomment-1455250495

**Reply by mikeboysen** [reply]
Thanks. I should have poked around first.
